src/leases/components/leaseSections/rent/RentAdjustmentEdit.js

```diff
@@ -341,7 +345,7 @@ class RentAdjustmentsEdit extends PureComponent<Props, State> {
       type,
       usersPermissions,
     } = this.props;
-    const {showSubventions} = this.state;
+    const showSubventions = typeof subventionType === "string"
```

Comment:
Miksi tyyppitarkistus eikä arvotarkistusta? Riittääkö, jos subventionType on asetettu eli se on jotain muuta kuin null tai undefined? Kelpaako myös tyhjä string?

Reply:
undefined tulee siinä tilanteessa, että alennus/korotus on poistettu. subventionType:n datatyyppi määrää sen, näytetäänkö subventiolistaus vai ei. Jos se on `null`, niin se piilotetaan. Jos se on `string`, oli se sitten tyhjä tai ei, niin sitten se näytetään. Tällä järjestelyllä subventiolistauksen näyttäminen pysyy synkassa Reduxin datan kanssa ja poistetaan irrallinen ja bugeja aiheuttanut `this.state.showSubventions`, jota pitäisi erikseen päivittää kun tietoja muutetaan.